app.rb

```diff
@@ -0,0 +1,90 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+require_relative './module'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+  include OtherFunctions
+
+  def create_person
+    selection =
+      take_input_label(
+        'Do you want to create a student (1) or a teacher (2)? [Input the number]'
+      )
+    selection = selection.to_i
+    case selection
+    when 1
+      student
+    when 2
+      teacher
+    else
+      put_lable('Invalid option')
+    end
+  end
+
+  def student
+    age = take_input_label('Age')
+    age = age.to_i
+    name = take_input_label('Name')
+    parent_permit = parent_permission
+    create_student(age, name, parent_permit)
+  end
+
+  def teacher
+    age = take_input_label('Age')
+    age = age.to_i
+    name = take_input_label('Name')
+    specialization = take_input_label('specialization')
+    create_teacher(age, name, specialization)
+  end
+
+  def list_people
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id} Age: #{person.age}"
+    end
+  end
+
+  def start_book
+    put_lable('Please enter the title of the book')
+    title = take_input_label('Title')
+    author = take_input_label('Author')
+    create_book(title, author)
+  end
+
+  def list_books
+    @books.each_with_index do |book, index|
+      puts "(#{index}) Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_rentals
+    id = take_input_label('Enter id of person')
+    @rentals.each do |rental|
+      put_lable('Rentals')
+      rental.person.id && id
+      put_lable("#{rental.date} - #{rental.book.title}")
+    end
+  end
```

Comment:
Great job, everything works fine 💯 except for the 'list_rentals' method when I try to enter a person's id it gives me an error, so Kindly fixt that 👍

https://user-images.githubusercontent.com/45231276/193787639-f1e25944-80e3-4998-a7e1-830b750773d4.mp4

Reply:
Done